conductr_cli/conduct_stop.py

```diff
@@ -12,16 +12,9 @@ def stop(args):
     """`conduct stop` command"""
 
     log = logging.getLogger(__name__)
-    path = 'bundles/{}?scale=0'.format(args.bundle)
-    url = conduct_url.url(path, args)
-    response = conduct_request.put(args.dcos_mode, conductr_host(args), url, auth=args.conductr_auth,
-                                   verify=args.server_verification_file, timeout=DEFAULT_HTTP_TIMEOUT)
-    validation.raise_for_status_inc_3xx(response)
-
-    if log.is_verbose_enabled():
-        log.verbose(validation.pretty_json(response.text))
+    response = stop_bundle(args)
 
-    response_json = json.loads(response.text)
+    response_json = json.loads(response)
```

Comment:
I think the `json.loads` should be done as part of `stop_bundle` since we're displaying the JSON response in the `stop_bundle` method if `log.is_verbose_enabled()`.